### ISSUE TITLE
Element is not clickable

### ISSUE THREAD

**Issue by phanisrikar** [SeleniumHQ/selenium#1969]
## Meta -
OS: 
Selenium Version: 
Browser: 
Browser Version: 

## Expected Behavior -
The element should be clickable .
## Actual Behavior -
The element is not being clickable and throwing an error 
Exception in performing operation Unable to perform click on  link 
 Element is not clickable at point (50, 366.3999938964844). Other element would receive the click: <div style="transition: all 600ms ease 0s; visibility: hidden;" id="page-spinner" class=" spinner-overlay"></div>
Command duration or timeout: 80 milliseconds
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'XXXXXXXXXXX', ip: '10.200.16.202', os.name: 'Windows Server 2008 R2', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_91'
Session ID: 4394eb74-ac8d-4de0-9e1e-1da9de455470
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=WINDOWS, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=45.0}] - By.linkText: Content Manager
20160414 03:00:57.063 :  INFO : 
buildnumber : nobuild
logDirName : c:\users\pchoda~1\appdata\local\temp\3\RIDEowoxfl.d\log.html
20160414 03:00:57.089 :  FAIL : 
Exception in performing  navigations Unable to perform click on  Manager link 
 Element is not clickable at point (50, 366.3999938964844). Other element would receive the click: <div style="transition: all 600ms ease 0s; visibility: hidden;" id="page-spinner" class=" spinner-overlay"></div>
Command duration or timeout: 80 milliseconds
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'XXXXXXXX', ip: 'xx.xx.xx.xx', os.name: 'Windows Server 2008 R2', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_91'
Session ID: 4394eb74-ac8d-4de0-9e1e-1da9de455470
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=WINDOWS, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=45.0}] - By.linkText: Content Manager

**Comment by p0deje** [comment]
Please provide [concise reproducible test case](http://sscce.org/) so that we could act on this issue.

**Comment by phanisrikar** [comment]
We are trying to click simple link using By.linkText("Content");
It was perfectly working when we were working on 2.46 version selenium and on firefox, chrome etc

But we updated the jar to 2.53 version and there are trouble clicking on majority of the clickable elements.

Is there any change related to click actions in the latest selenium version.?

We tried few alternatives like using xpath and css but the behavior is not consistent and the case fails very frequently.

We also tried on different machines and browser version as well like FF44, 45 and chrome latest and even in mac.

every case was working perfectly with older selenium version.

Any info on the clickable elements is very helpful

Thanks
